pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -116,6 +119,39 @@ func TestShowChangefeedJobsBasic(t *testing.T) {
 	cdcTest(t, testFn, feedTestOmitSinks("webhook", "sinkless"), feedTestNoExternalConnection)
 }
 
+func TestShowChangefeedJobsShowsHighWaterTimestamp(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	testFn := func(t *testing.T, s TestServer, f cdctest.TestFeedFactory) {
+		sqlDB := sqlutils.MakeSQLRunner(s.DB)
+		sqlDB.Exec(t, `CREATE TABLE foo(a INT PRIMARY KEY)`)
+		foo := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='0s',min_checkpoint_frequency='0s'`)
+
+		defer closeFeed(t, foo)
+
+		var highWaterTimestamp gosql.NullFloat64
+		var readableHighWaterTimestamp gosql.NullTime
+
+		// Wait for the high water timestamp to be non-null.
+		testutils.SucceedsSoon(t, func() error {
+			sqlDB.QueryRow(t, "SELECT high_water_timestamp, readable_high_water_timestamp from [SHOW CHANGEFEED JOBS]").Scan(&highWaterTimestamp, &readableHighWaterTimestamp)
```

Comment:
nit: this is a very long line, the style guide recommends wrapping comments at 80 characters and code at 100 characters https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/181371303/Go+Golang+coding+guidelines#Line-Length